CHANGELOG.md

```diff
@@ -1,3 +1,65 @@
+## 1.6.0 / 2017-04-14
+
+* [CHANGE] Replaced the **experimental** remote write implementations for
+  various backends by a generic write interface with example bridge
+  implementation for various backends.
+* [FEATURE] New flag `-storage.local.target-heap-size` to tell Prometheus about
+  the desired heap size. This deprecates the flags
+  `-storage.local.memory-chunks` and `-storage.local.max-chunks-to-persist`,
+  which are kept for backward compatibility.
+* [FEATURE] Add `check-metrics` to `promtool` to lint metric names.
+* [FEATURE] Auto-tweak times between checkpoints to limit time spent in
```

Comment:
I was tempted to declare it a `[CHANGE]` (as it will create less frequent checkpoints, even in scenarios where users might desire checkpoints as frequently as possible), but then it would require us to call it 2.0... ;o)

One person's bugfix is another person's breaking change...

If two of you are happy with burying this as an enhancement, so am I.